src/app/workspace/resource/representation/video/video.component.ts

```diff
@@ -340,6 +386,24 @@ export class VideoComponent implements OnInit, AfterViewInit {
         this.previewTime = this.previewTime < 0 ? 0 : this.previewTime;
     }
 
+    private _getOriginalFilename() {
+        const requestOptions = {
+            headers: new HttpHeaders({ 'Content-Type': 'application/json' }),
+            withCredentials: true
+        };
+
+        const index = this.src.fileValue.fileUrl.indexOf(this.src.fileValue.filename);
+        const pathToJson = this.src.fileValue.fileUrl.substring(0, index + this.src.fileValue.filename.length) + '/knora.json';
+
+        this._http.get(pathToJson, requestOptions).subscribe(
+            res => {
+                console.log('a', res);
```

Comment:
these console.logs can be removed

Reply:
fixed in https://github.com/dasch-swiss/dsp-app/pull/798/commits/b8527777bdcee6ca0fe2370ec4ea743cf1e193f5